Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM golang:1.11-alpine AS build
-RUN apk add --update make
+FROM golang:1.12-alpine AS build
+RUN apk add --update make && apk add --no-cache git
```

Comment:
FYI, we could probably do everything in one command. 
```diff
- RUN apk add --update make && apk add --no-cache git
+ RUN apk add --update --no-cache make git
```
Nothing to change now though.